webapp-django/crashstats/manage/static/manage/js/api_tokens.js

```diff
@@ -0,0 +1,228 @@
+/*global alert:true location:true */
+(function ($, document) {
+    'use strict';
+
+    function start_loading() {
+        $('.pleasewait').show();
+    }
+
+    function stop_loading() {
+        $('.pleasewait').hide();
+    }
+
+    function fetch() {
+
+        function show_pagination_links(count, batch_size, page) {
```

Comment:
You're creating more work for me :(
You unfortunately you're right. I'll do some refactoring.